pyproject.toml

```diff
@@ -57,6 +57,8 @@ dev = [
     "awkward-pandas;python_version >= \"3.8\"",
 ]
 test = [
+    "aiohttp",
+    "fsspec-xrootd",
```

Comment:
Should we keep only `fsspect-xrootd` or should we also add `fsspect` explicitly? `fsspec` is included inside `fsspect-xrootd`. Just wondering what would be best practise.

Reply:
The usual practice is - if you import anything from package X, declare it in your dependencies, even if package Y requires X. Otherwise, package Y could change its dependency in future, and your package would break! :)

Reply:
This was discussed a bit in the last PR https://github.com/scikit-hep/uproot5/pull/692#issuecomment-1227709118 basically to avoid having an unnecessarily long chain of `install this` errors when using for the first time.